spec/javascripts/health-care/components/FinancialAssessmentPanel.spec.jsx

```diff
@@ -1,14 +0,0 @@
-import React from 'react';
```

Comment:
Do we need to create specs now for the sections or is this not really necessary to test anymore?